### PR TITLE
Open group

### PR DESCRIPTION
**Pull request by jum-s**
linked to [#460 open group server branch](https://github.com/inventaire/inventaire/pull/460)

### CODE REVIEW

**Review by maxlath — DISMISSED** [review]

File: app/modules/network/lib/group_form_data.js
Comment (jum-s) [reply app/modules/network/lib/group_form_data.js:22]: shorter suggestion as there already are warnings

```suggestion
      label: 'no admin validation required'
```
?

File: app/modules/network/views/templates/group_board_header.hbs
Comment (jum-s) [reply app/modules/network/views/templates/group_board_header.hbs:9]: sure, ill apply the suggestion.
Although all those cryptic icons exist with the sole purpose of a having a alt text defined that can be hovered

File: app/modules/network/views/templates/group_board_header.hbs
Comment (jum-s) [reply app/modules/network/views/templates/group_board_header.hbs:13]: again icons are too much subjected to interpretation, focusing on a clear text is preferable

File: app/modules/network/views/templates/open.hbs
Comment (jum-s) [reply app/modules/network/views/templates/open.hbs:5]: yep

File: app/modules/network/views/templates/group_board_header.hbs
Comment (jum-s) [reply app/modules/network/views/templates/group_board_header.hbs:9]: what i mean by the precendent comment, explain by a brillant text [FR] https://lesamisdebartleby.wordpress.com/2019/06/10/ivan-illich-lage-des-systemes/

**Comment by jum-s** [comment]
rebased and autosquashed

File: app/modules/network/lib/group_form_data.js
Comment (jum-s) [reply app/modules/network/lib/group_form_data.js:22]: i think its fine to have a short name like open membership as long as there is a clear explanation right below

**Comment by maxlath** [comment]
before merging this into master, we might want to wait for this issue to be solved https://github.com/inventaire/inventaire/issues/471 as that would probably mean changing the group settings again, meaning potentially creating more confusion

**Comment by jum-s** [comment]
not sure of how to measure the confusion you're talking about (how is changing several things at once less confusing than small iterative changes?). What is sure is the mental charge of maintainers if we delay a ready-to-merge branch (remember this branch exists, make sure of rebasing #447 on this branch, etc.)

**Comment by maxlath** [comment]
you say it's ready-to-merge, but different [concerns expressed here](https://github.com/inventaire/inventaire/issues/456#issuecomment-729921968) remain unaddressed. I have started working on the point on notifications (see [`open-group` server branch](https://github.com/inventaire/inventaire//tree/open-group)).

I value not letting PRs hang around too long, but think unease and resistance of other maintainers, even if worded poorly or of an intuitive nature, should be respected and have priority over that principle: if resistance are raised, we should take our time rather than rush things, that's the sense of having [unanimity as our default deliberation principle](https://wiki.inventaire.io/wiki/Statuses)

**Comment by jum-s** [comment]
sure, i just didnt have those comments in mind. You were making a very different point in your last comment that i was addressing, that's it.

**Comment by jum-s** [comment]
points that are neither inintuitive nor poorly written btw

**Comment by jum-s** [comment]
TODO post-merge :
 - [ ] replace i18n `leave_group_warning` from `this can be undone later by requesting to join the group again, but you will need an admin's approval` to `this can be undone later by requesting to join the group again`

**Pull request merged**
